### PR TITLE
Export empty vt_gen target (for standalone builds)

### PR DESCRIPTION
**Pull request by etcwilde**
Fixing the standalone builds by exporting the vt_gen target so that the sources of the clangCodeGen target can form an explicit dependency edge on it.

### CODE REVIEW

**Comment by etcwilde** [comment]
CC @RossComputerGuy

File: llvm/cmake/modules/CMakeLists.txt [hunk 1]
```diff
@@ -33,6 +33,9 @@ endforeach(lib)
 if(intrinsics_gen IN_LIST LLVM_COMMON_DEPENDS)
   list(REMOVE_ITEM LLVM_COMMON_DEPENDS intrinsics_gen)
 endif()
+if(vt_gen IN_LIST LLVM_COMMON_DEPENDS)
```
Comment (compnerd) [inline llvm/cmake/modules/CMakeLists.txt:36]: Lexicographical ordering?

File: llvm/cmake/modules/LLVMConfig.cmake.in [hunk 2]
```diff
@@ -151,6 +151,9 @@ endif()
 if(NOT TARGET intrinsics_gen)
   add_custom_target(intrinsics_gen)
 endif()
+if(NOT TARGET vt_gen)
```
Comment (compnerd) [inline llvm/cmake/modules/LLVMConfig.cmake.in:154]: Likewise

Comment (etcwilde) [reply llvm/cmake/modules/LLVMConfig.cmake.in:154]: Do you want me to re-order all of them? `acc_gen` comes after `omp_gen` and `intrinsics_gen` in this list.

Comment (compnerd) [reply llvm/cmake/modules/LLVMConfig.cmake.in:154]: Ugh, leave this one be I suppose.

**Review by mgorny — APPROVED** [review]
This works too. Thanks!

**Review by compnerd — APPROVED** [review]

**Comment by RossComputerGuy** [comment]
Will give this a try later today when I have the time.

**Comment by RossComputerGuy** [comment]
Sweet, this worked in standalone (nix).

**Pull request merged**